### PR TITLE
Improve documentation quality - Bunch #1

### PR DESCRIPTION
**Pull request by cristobal-buenrostro**
Verify assignments pages in order to improve the quality in doc;
Aspects to consider are:
- beta & v1.0 matching
- Properties descriptions
- Response/request bodies having properties with correct values.

### CODE REVIEW

File: api-reference/v1.0/api/educationassignment-setupresourcesfolder.md [hunk 1]
```diff
@@ -26,9 +26,8 @@ One of the following permissions is required to call this API. To learn more, in
 
 ## HTTP request
 <!-- { "blockType": "ignored" } -->
-
 ```http
-POST /education/classes/acdefc6b-2dc6-4e71-b1e9-6d9810ab1793/assignments/cf6005fc-9e13-44a2-a6ac-a53322006454/setUpResourcesFolder
+POST /education/classes/{id}/assignments/{id}/setUpResourcesFolder
```
Comment (Lauragra) [inline api-reference/v1.0/api/educationassignment-setupresourcesfolder.md:30]: Why are we putting placeholders in this example? It should use the same values shown in the beta version of the topic, right?

File: api-reference/v1.0/resources/educationresource.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "educationResource resource type"
-description: "A superclass for all resource objects in the system."
+description: "A superclass for all resource objects in the system. A resource is associated with an **Assignment** and/or **Submission**, which represents the learning object that is being"
```
Comment (Lauragra) [inline api-reference/v1.0/resources/educationresource.md:3]: The second sentence is incomplete. We should keep the descriptions to a single sentence, if possible. The longer description can be in the summary of the topic.

File: api-reference/v1.0/resources/educationsubmission.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "educationSubmission resource type"
-description: "A submission represents the resources that an individual (or group) turn in for an assignment and the outcomes (such as grades or feedback) that are associated with the submission."
+description: "Submissions are owned by an assignment. A submission represents the resources that an individual (or group) turn in for an assignment and the outcomes (such as grades or feedback) that are associated with the submission."
```
Comment (Lauragra) [inline api-reference/v1.0/resources/educationsubmission.md:3]: Please remove this. The description field should be a high-level summary description that can show up in search so readers can get a quick glance of the content.

File: api-reference/v1.0/api/educationassignment-setupresourcesfolder.md (hunk 1 above)
Comment (cristobal-buenrostro) [reply api-reference/v1.0/api/educationassignment-setupresourcesfolder.md:30]: beta is using the placeholders too, I just changed it to make it matching with beta.

The values are used but in the Request Body example.

File: api-reference/v1.0/resources/educationresource.md (hunk 2 above)
Comment (cristobal-buenrostro) [reply api-reference/v1.0/resources/educationresource.md]: done

File: api-reference/v1.0/resources/educationsubmission.md (hunk 3 above)
Comment (cristobal-buenrostro) [reply api-reference/v1.0/resources/educationsubmission.md]: done, removed and same description was applied to beta.

File: api-reference/beta/resources/educationoutcome.md [hunk 4]
```diff
@@ -25,6 +25,13 @@ The result of grading an assignment. This is a base class; the derived types are
 
 None
 
+## Properties
+| Property	   | Type	|Description|
+|:---------------|:--------|:----------|
+|id|String|Read-only.|
+|lastModifiedBy|[identitySet](identityset.md)|The individual who updated the resource.|
+|lastModifiedDateTime|DateTimeOffset|Moment in time when the resource was last modified.  The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2021 is `2021-01-01T00:00:00Z`.|
+
```
Comment (isvargasmsft) [inline api-reference/beta/resources/educationoutcome.md:34]: Please move the Properties section between the Methods and Relationships. Properties should always go after Methods.

File: api-reference/v1.0/resources/educationoutcome.md [hunk 5]
```diff
@@ -25,6 +25,13 @@ This is a base class; the derived types are [educationFeedbackOutcome](education
 
 None
 
+## Properties
+| Property	   | Type	|Description|
+|:---------------|:--------|:----------|
+|id|String|Read-only.|
+|lastModifiedBy|[identitySet](identityset.md)|The individual who updated the resource.|
+|lastModifiedDateTime|DateTimeOffset|Moment in time when the resource was last modified.  The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2021 is `2021-01-01T00:00:00Z`.|
+
```
Comment (isvargasmsft) [inline api-reference/v1.0/resources/educationoutcome.md:34]: Please move this section after Methods.

File: api-reference/beta/resources/educationoutcome.md (hunk 4 above)
Comment (cristobal-buenrostro) [reply api-reference/beta/resources/educationoutcome.md:30]: done

File: api-reference/v1.0/resources/educationoutcome.md (hunk 5 above)
Comment (cristobal-buenrostro) [reply api-reference/v1.0/resources/educationoutcome.md:30]: done

**Review by isvargasmsft — APPROVED** [review]

**Comment by cristobal-buenrostro** [comment]
The actions below have been taken in order to include an additional change to solve a customer issue related to setUpResourceFolder article.

- added the **do not merge** tag
- added the **ready for content review** tag
- removed **ready to merge** tag
- linked the customer issue #13731

File: api-reference/beta/api/educationassignment-setupresourcesfolder.md [hunk 6]
```diff
@@ -143,6 +144,33 @@ Content-length: 279
 }
 ```
 
+If this method was called before, then it returns a `400 Bad request` and an error response.
```
Comment (cristobal-buenrostro) [inline api-reference/beta/api/educationassignment-setupresourcesfolder.md:147]: This response example was added (beta&v1.0) in order to attend customer issue.

Comment (isvargasmsft) [reply api-reference/beta/api/educationassignment-setupresourcesfolder.md:147]: 1. Please rephrase to: If the specified assignment already has a folder, this method returns a `400 Bad request` and an error response. 
2. Please include just this line in the Request body section, after line 43.

Comment (cristobal-buenrostro) [reply api-reference/beta/api/educationassignment-setupresourcesfolder.md]: done

**Review by isvargasmsft — APPROVED** [review]

**Pull request merged**